src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceMainTabClassic.java

```diff
@@ -0,0 +1,100 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * SalesForceMainTabClassic.java
+ * Class of Salesforce main tab that contains all the options.
+ */
+public class SalesForceMainTabClassic {
+    private WebDriver webDriver;
+    private By tabButton;
+    private By accountsLink;
+    private WebDriverWait wait;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "wait" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)